common/src/main/java/org/apache/drill/common/exceptions/UserException.java

```diff
@@ -37,6 +37,7 @@
  *
  * @see org.apache.drill.exec.proto.UserBitShared.DrillPBError.ErrorType
  */
+@SuppressWarnings("serial")
```

Comment:
There are three ways to handle serializable types. 1) assign a number (start with 1 and increase, say). 2. Let an IDE generate one, such as the one you mentioned. or 3) state that the class is never serialized, so don't worry about the ID, which is the approach I took. Drill never uses Java serialization so the serial ID is unused and unnecessary.